backend-diagram-converter/core/src/main/java/org/camunda/community/migration/converter/expression/ExpressionTransformer.java

```diff
@@ -75,4 +75,16 @@ private String handleExpression(String expression) {
     }
     return replaced;
   }
+
+  public static boolean hasMethodInvocation(String expression) {
+    Pattern pattern = Pattern.compile("[^\\s]\\.get.*\\(\\)");
+    Matcher m = pattern.matcher(expression);
+    return m.find();
+  }
+
+  public static boolean hasExecution(String expression) {
```

Comment:
The `ExpressionTransformationResult` could contain information like this

Reply:
Same here. Check is now with `transformationResult.hasExecution()`.